### PR TITLE
Dom_html: adding hidden, onfullscreenchange and onwebkitfullscreenchange into 'document' (3.10.1)

### PR DESCRIPTION
**Pull request by Thibaut-Gudin**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # ?? (??) - ??
 
+# 3.10.1 (2021-09-TODO) - Paris
```
Comment (jrochel) [inline CHANGES.md:3]: TODO

**Review by jrochel — CHANGES_REQUESTED** [review]

File: CHANGES.md (hunk 1 above)
Comment (Thibaut-Gudin) [reply CHANGES.md]: I completely forgot about it indeed, it should be good now I hope.

File: CHANGES.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # ?? (??) - ??
 
+# 3.10.1 (2021-09-06) - Paris
```
Comment (hhugo) [inline CHANGES.md:3]: Why do we have a version here ? has there been a new release ?

Comment (jrochel) [reply CHANGES.md:3]: Sorry, my bad. This is a version number used for developement. I haven't paid close enough attention to the versioning practices of js_of_ocaml. (I'm used to projects having an opam file in the repository with a version that changes from PR to PR.)

Comment (jrochel) [reply CHANGES.md:3]: https://github.com/ocsigen/js_of_ocaml/pull/1128

**Comment by hhugo** [comment]
@jrochel, I usually "squash and merge" GPR to keep the git history clean. I occasionally "rebase and merge" if I individual commits are relevant.
I would avoid "create a merge commit"

**Comment by jrochel** [comment]
Noted.

**Pull request merged**